docs/docs/getting-started/install.md

```diff
@@ -92,6 +92,31 @@ curl -LO https://github.com/edgelesssys/constellation/releases/latest/download/c
 sudo install constellation-darwin-amd64 /usr/local/bin/constellation
 ```
 
+</tabItem>
+
+<tabItem value="windows-amd64" label="Windows (amd64)">
+
+1. Download the CLI:
+
+```bash
+Invoke-WebRequest -OutFile ./constellation.exe -Uri 'https://github.com/edgelesssys/constellation/releases/latest/download/constellation-windows-amd64.exe'
```

Comment:
```suggestion
Invoke-WebRequest -OutFile ( New-Item -Path 'C:\Program Files\Constellation\bin\constellation.exe' -Force ) -Uri 'https://github.com/edgelesssys/constellation/releases/latest/download/constellation-windows-amd64.exe'
```
Allows you to skip step 3, but might require admin privileges when starting powershell.

Reply:
I think starting powershell as admin might trigger peoples intrusion detection. Let's keep it like this.